libs/sdk-ui-filters/src/AttributeFilter/AttributeDropdown/AttributeDropdownBody.tsx

```diff
@@ -6,8 +6,30 @@ import { AttributeDropdownList } from "./AttributeDropdownList";
 import { AttributeDropdownButtons } from "./AttributeDropdownButtons";
 import { AttributeListItem } from "./types";
 import { AttributeDropdownItemsFilteredBody } from "./AttributeDropdownItemsFilteredBody";
+import { ILegacyMultiSelectListItemProps } from "@gooddata/sdk-ui-kit";
+import { ObjRef } from "@gooddata/sdk-model";
 
-interface IAttributeDropdownBodyProps {
+/**
+ * @public
+ */
+export interface IAttributeDropdownBodyExtendedProps extends IAttributeDropdownBodyProps {
+    deleteFilter?: () => void;
+    isLoaded: boolean;
+    isElementsLoading: boolean;
+    width?: number;
+    listItemClass?: React.ComponentType<ILegacyMultiSelectListItemProps>;
```

Comment:
This is problematic: `ILegacyMultiSelectListItemProps` is internal and can change at any time. We need to duplicate this type here unfortunately.